test/sol6/kyberNetwork.js

```diff
@@ -1207,6 +1202,28 @@ contract('KyberNetwork', function(accounts) {
                 for (let i = 0; i < expectedResult.e2tIds.length; i++) {
                     Helper.assertEqual(expectedResult.e2tIds[i], actualE2Tids[i], "E2T id not equal");
                 }
+
+                let actualT2EsrcAmounts = txResult.logs[3].args.t2eSrcAmounts;
+                let actualE2TsrcAmounts = txResult.logs[3].args.e2tSrcAmounts;
+                Helper.assertEqual(expectedResult.t2eSrcAmts.length, actualT2EsrcAmounts.length, "T2E srcAmounts length not equal");
```

Comment:
Helper.assertArrayEqual(expectedResult.t2eSrcAmts, actualT2EsrcAmounts);
Helper.assertArrayEqual(expectedResult.e2tSrcAmts, actualE2TsrcAmounts);

Reply:
Why do we use "Amts" and "Amounts" to express the same meaning of "amount"?

Reply:
I think we should change Amt --> Amounts.
its plural here since its an array.

Reply:
Amts changed to Amounts. Commit still squashed to one.